UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -166,20 +166,26 @@ public enum RoomEntryFlags : uint
         /// </summary>
         public UndertaleSimpleList<UndertaleResourceById<UndertaleSequence, UndertaleChunkSEQN>> Sequences { get; private set; } = new UndertaleSimpleList<UndertaleResourceById<UndertaleSequence, UndertaleChunkSEQN>>();
 
-        private Layer GetBGColorLayer()
-        {
-            return _layers?.Where(l => l.LayerType is LayerType.Background
-                                    && l.BackgroundData.Sprite is null
-                                    && l.BackgroundData.Color != 0)
-                           .OrderBy(l => l?.LayerDepth ?? 0)
-                           .FirstOrDefault();
-        }
         public void UpdateBGColorLayer() => OnPropertyChanged("BGColorLayer");
 
         /// <summary>
-        /// The layer containing the background color.
+        /// The layer containing the background color.<br/>
         /// </summary>
-        public Layer BGColorLayer => GetBGColorLayer();
+        /// <remarks>
+        /// Used by "BGColorConverter" of the UndertaleModTool room editor.
+        /// This attribute is UMT-only and does not exist in GameMaker.
+        /// </remarks>
+        public Layer BGColorLayer
+        {
+            get
+            {
+                return _layers?.Where(l => l.LayerType is LayerType.Background
+                                        && l.BackgroundData.Sprite is null
+                                        && l.BackgroundData.Color != 0)
+                               .OrderBy(l => l?.LayerDepth ?? 0)
```

Comment:
```suggestion
                               .OrderBy(l => l.LayerDepth ?? 0)
```
the layers are known to not be null there.